### PR TITLE
docs(cn): fix the format of some files

### PR DESCRIPTION
**Pull request by jacob-lcs**
_describe your changes..._

 `src/content/guides/getting-started.md` 章节中的一段没有翻译

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

File: src/content/guides/production.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ T> 以下示例来源于 [tree shaking](/guides/tree-shaking) 和 [开发环境]
 
 **development(开发环境)** 和 **production(生产环境)** 这两个环境下的构建目标存在着巨大差异。在**开发环境**中，我们需要：强大的 source map 和一个有着 live reloading(实时重新加载) 或 hot module replacement(热模块替换) 能力的 localhost server。而**生产环境**目标则转移至其他方面，关注点在于压缩 bundle、更轻量的 source map、资源优化等，通过这些优化方式改善加载时间。由于要遵循逻辑分离，我们通常建议为每个环境编写**彼此独立的 webpack 配置**。
 
-虽然，以上我们将_生产环境_和_开发环境_做了略微区分，但是，请注意，我们还是会遵循不重复原则(Don't repeat yourself - DRY)，保留一个 "common(通用)" 配置。为了将这些配置合并在一起，我们将使用一个名为 [`webpack-merge`](https://github.com/survivejs/webpack-merge) 的工具。此工具会引用 "common" 配置，因此我们不必再在环境特定(environment-specific)的配置中编写重复代码。
+虽然，以上我们将 _生产环境_ 和 _开发环境_ 做了略微区分，但是，请注意，我们还是会遵循不重复原则(Don't repeat yourself - DRY)，保留一个 "common(通用)" 配置。为了将这些配置合并在一起，我们将使用一个名为 [`webpack-merge`](https://github.com/survivejs/webpack-merge) 的工具。此工具会引用 "common" 配置，因此我们不必再在环境特定(environment-specific)的配置中编写重复代码。
```
Comment (QC-L) [inline src/content/guides/production.md]:
```suggestion
虽然，以上我们将 _生产环境_ 和 _开发环境_ 做了细微区分，但是，请注意，我们还是会遵循不重复原则(Don't repeat yourself - DRY)，保留一个 "common(通用)" 配置。为了将这些配置合并在一起，我们将使用一个名为 [`webpack-merge`](https://github.com/survivejs/webpack-merge) 的工具。此工具会引用 "common" 配置，因此我们不必再在环境特定(environment-specific)的配置中编写重复代码。
```

File: src/content/guides/getting-started.md [hunk 2]
```diff
@@ -314,7 +314,7 @@ cacheable modules 530 KiB
 webpack 5.4.0 compiled successfully in 1940 ms
 ```
 
-T> Custom parameters can be passed to webpack by adding two dashes between the `npm run build` command and your parameters, e.g. `npm run build -- --color`.
+T> 可以通过在 `npm run build` 命令与参数之间添加两个短划线的方式向 webpack 传递自定义参数，例如：`npm run build -- --color`。
```
Comment (QC-L) [inline src/content/guides/getting-started.md]:
```suggestion
T> 可以通过在 `npm run build` 命令与参数之间添加两个连接符的方式向 webpack 传递自定义参数，例如：`npm run build -- --color`。
```

File: src/content/loaders/css-loader.md [hunk 3]
```diff
@@ -1135,7 +1135,7 @@ module.exports = {
 
 如下配置的 `webpack.config.js` 可以加载 CSS 文件，嵌入小的 PNG/JPG/GIF/SVG 图片以及字体作为[数据 URL](https://tools.ietf.org/html/rfc2397)，并将较大的文件复制到输出目录。
 
-**For webpack v5:**
+**对于 webpack v5:**
```
Comment (QC-L) [inline src/content/loaders/css-loader.md]:
```suggestion
**对于 webpack v5：**
```

File: src/content/loaders/css-loader.md [hunk 4]
```diff
@@ -1149,15 +1149,15 @@ module.exports = {
...
+        // 更多信息请点击这里 https://webpack.js.org/guides/asset-modules/
         type: "asset",
       },
     ],
   },
 };
 ```
 
-**For webpack v4:**
+**对于 webpack v4:**
```
Comment (QC-L) [inline src/content/loaders/css-loader.md]:
```suggestion
**对于 webpack v4：**
```

File: src/content/loaders/less-loader.md [hunk 5]
```diff
@@ -422,19 +422,17 @@ module.exports = {
 
 首先我们会尝试使用内置 `less` 解析逻辑，然后再使用 `webpack` 解析逻辑（alias 和 `~`）。
 
-#### Webpack 解析器 {#webpack-resolver}
+#### webpack 解析器 {#webpack-resolver}
 
-`webpack` 提供了一种 [解析文件的高级机制](/configuration/resolve/)。
-如果 `less` 不能解析 `@import` 的话，`less-loader` 作为 Less 的插件将所有的查询结果传递给 webpack 解析器。
-因此你可以从 `node_modules` 中导入 Less 模块。
+`webpack` 提供了一种 [解析文件的高级机制](/configuration/resolve/)。如果 `less` 不能解析 `@import` 的话，`less-loader` 作为 Less 的插件将所有的查询结果传递给 webpack 解析器。因此你可以从 `node_modules` 中导入 Less 模块。
```
Comment (QC-L) [inline src/content/loaders/less-loader.md:427]: 这里没按行数处理

Comment (jacob-lcs) [reply src/content/loaders/less-loader.md:427]: 这里应该是当时处理冲突的时候没有处理掉，目前[英文文档](https://www.npmjs.com/package/less-loader)是当做**一行**处理的

**Review by QC-L — APPROVED** [review]

**Pull request merged**